app/src/main/java/com/duckduckgo/app/browser/WebViewRequestInterceptor.kt

```diff
@@ -248,7 +248,7 @@ class WebViewRequestInterceptor(
                     documentUrl = documentUrl,
                     feed = isMalicious.feed,
                     exempted = false,
-                    clientSideHit = false,
+                    clientSideHit = true,
```

Comment:
should this be `result.clientSideHit`? Otherwise seems we are overriding what was received.

Reply:
No. Since this method is `handleConfirmationCallback`, we know we've hit the server